src/android/android-uvc.h

```diff
@@ -0,0 +1,105 @@
+        {
+        public:
+            android_uvc_device(const uvc_device_info& info, std::shared_ptr<const android_backend> backend);
+            ~android_uvc_device();
+
+            void probe_and_commit(stream_profile profile, frame_callback callback, int buffers) override;
+
+            // open thread and take frames
+            void stream_on(std::function<void(const notification& n)> error_handler = [](const notification& n){}) override;
+
+            void start_callbacks() override;
+            void stop_callbacks() override;
+            void close(stream_profile profile) override;
+            void set_power_state(power_state state) override;
+            power_state get_power_state() const override { return _power_state; }
+            std::vector<stream_profile> get_profiles() const override;
+
+            static bool is_connected(const uvc_device_info& info);
+
+            void init_xu(const extension_unit& xu) override;
+            bool set_xu(const extension_unit& xu, uint8_t ctrl, const uint8_t* data, int len) override;
+            bool get_xu(const extension_unit& xu, uint8_t ctrl, uint8_t* data, int len) const override;
+            control_range get_xu_range(const extension_unit& xu, uint8_t ctrl, int len) const override;
+
+            bool get_pu(rs2_option opt, int32_t& value) const override;
+            bool set_pu(rs2_option opt, int value) override;
+            control_range get_pu_range(rs2_option opt) const override;
+
+            void lock() const override { _systemwide_lock.lock(); }
+            void unlock() const override {_systemwide_lock.unlock(); }
+
+            std::string get_device_location() const override { return _location; }
+            usb_spec get_usb_specification() const override { return _info.conn_spec; }
+
+        private:
+            friend class source_reader_callback;
+
+            int32_t rs2_value_translate(uvc_req_code action, rs2_option option, int32_t value) const;
+            void play_profile(stream_profile profile, frame_callback callback);
+            void stop_stream_cleanup(const stream_profile& profile, std::vector<profile_and_callback>::iterator& elem);
+            void flush(int sIndex);
+            void check_connection() const;
+
+            int rs2_option_to_ctrl_selector(rs2_option option, int &unit) const;
+            int32_t get_data_usb(uvc_req_code action, int control, int unit, unsigned int length = sizeof(uint32_t)) const;
+            void set_data_usb(uvc_req_code action, int control, int unit, int value) const;
+
+            const uvc_device_info                   _info;
+            power_state                             _power_state = D3; // power state change is unsupported
+
+            uint16_t                                _streamIndex;
+            std::vector<profile_and_callback>       _streams;
+            std::mutex                              _streams_mutex;
+
+            std::shared_ptr<const android_backend>      _backend;
```

Comment:
Can we get `_backend` in line with the rest?

Reply:
There were 3 ints further down that didn't align either. Now none of it aligns - it's a waste of white space. b17aa05a0